### PR TITLE
sys-auth/fprintd: version bump

### PR DESCRIPTION
**Pull request by nephros**
- fix SRC_URI
- fix dependency on sys-auth/libfprint <= 1.90
- patch configure.ac to respect a --disable-systemd option
- introduce systemd USE flag
- use it to enable compiling on non-systemd systems
- bump version 0.9.0

Closes: https://bugs.gentoo.org/704158
Closes: https://bugs.gentoo.org/700282

Signed-off-by: Peter Gantner <gentoo@nephros.org>

### CODE REVIEW

**Comment by nephros** [comment]
NOTE: the --enable/disable-systemd switch needs testing, I don't have a system where I can do that.

File: sys-auth/fprintd/fprintd-0.8.1-r2.ebuild [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools pam systemd
+
+DESCRIPTION="D-Bus service to access fingerprint readers"
+HOMEPAGE="https://cgit.freedesktop.org/libfprint/fprintd/"
```
Comment (charIes17) [inline sys-auth/fprintd/fprintd-0.8.1-r2.ebuild:9]: No sure, but what about https://fprint.freedesktop.org/ ?

**Comment by bearsh** [comment]
this PR seems to be outdated and superseded by 7c1b363ca1aecbe5c0c878d5d6583d869b01d073. can someone with the appropriate privilege please close this?

**Comment by nephros** [comment]
> this PR seems to be outdated and superseded by [7c1b363](https://github.com/gentoo/gentoo/commit/7c1b363ca1aecbe5c0c878d5d6583d869b01d073). can someone with the appropriate privilege please close this?

OK, closing.

**Pull request closed without merging**